data/chameleoncloud/sites/tacc/clusters/chameleon/nodes/0e9a516d-8cb4-49fd-810b-74fee5c186d1.json

```diff
@@ -5,51 +5,62 @@
     "smt_size": 48
   },
   "bios": {
-    "release_date": "Jan 13 2022",
-    "vendor": "FUJITSU",
+    "release_date": "2022-01-13",
+    "vendor": "Fujitsu",
     "version": "1.9.0"
   },
   "chassis": {
-    "manufacturer": "FUJITSU",
-    "name": "FX700"
+    "manufacturer": "Fujitsu",
+    "name": "PowerEdge FX700",
+    "serial": "-"
```

Comment:
this might be a parsing error?

Reply:
when I do a `data save` on that one I get:
```
    "system_vendor": {
      "product_name": "FX700 (-)",
      "serial_number": "-",
      "manufacturer": "FUJITSU"
    },
```